qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx

```diff
@@ -45,22 +51,35 @@ export const IssueTabView = component$(({ activeTab, owner, name }: IssuesProps)
     activeTab: activeTab,
   });
   const dropdownStore = useStore<DropdownStoresProps>({
-    selectedLabel: labelOptions[0].value,
+    selectedLabel: undefined,
     selectedSort: sortOptions[0].value,
-    selectedMilestones: milestonesOptions[0].value,
+    selectedMilestones: undefined,
```

Comment:
why undefined and not null?

Reply:
in order to have the first "no milestone selected" value checked when we open the filter dropdown. We check the value of the option with the dropdown store value and the initial value of the store has no "selectedMilestone" so is computed as undefined, not null. (Don't know if I was clear, sorry)